virt/virt-4-12-release-notes.adoc

```diff
@@ -182,9 +146,29 @@ Deprecated features are included in the current release and supported. However,
 [id="virt-4-12-removed"]
 === Removed features
 
+Removed features are not supported in the current release.
 
-[id="virt-4-12-changes"]
-== Notable technical changes
+//CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} 4.11 removed support for link:https://nmstate.io/[nmstate], including the following objects:
```

Comment:
A little confused here: Is this content for 4.11 or 4.12? Just making sure.

Reply:
It was removed in 4.11, we're leaving in for instructions when upgrading in case folks are a release behind.